cmd/daemon.go

```diff
@@ -128,6 +131,9 @@ var tokenCmd = &cobra.Command{
 }
 
 func init() {
+	if os.Getenv("DAEMON") != "true" {
+		daemonCmd.Hidden = true
```

Comment:
Interesting

Reply:
It shows up with `inertia --help` - figured we probably don't want people using the `daemon` commands at home 😆